.github/ISSUE_TEMPLATE/Update A-enviar-software-para-revision.md

```diff
@@ -0,0 +1,105 @@
+---
+name: Enviar software para revisión
+about: Usa esta plantilla para enviar un software a revisar en Español (Experimental)
+---
+
+
+Autor Encargado: nombre (@github_handle)  
+Repositorio:   
+Versión Enviada:   
+Editor: TBD  
+Revisor 1: TBD  
+Revisor 2: TBD  
+Archivo: TBD  
+Versión Aceptada: TBD   
+
+---
+
+
+
+-   Pega el archivo DESCRIPTION completo dentro del siguiente bloque de código.
+
+```
+
+```
+
+
+## Alcance 
+
+- Por favor, indica qué categoria(s) aplican a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes (Inglés=](https://ropensci.github.io/dev_guide/policies.html#package-categories). Por favor, tilda todas las apropiadas. Si no estás seguro, te sugerimos que comiences un pre-envío.
+
+	- [ ] recuperación de datos (data retrieval)
+	- [ ] extracción de datos (data extraction)
+	- [ ] munging de datos (data munging)
```

Comment:
Is "munging" accepted as a word in Spanish? If not, maybe use quotes or italics, e.g. "munging" or _munging_ (like we do when we use Latin words).

Reply:
I like that the name in English are between parentheses, it is nice for finding definitions in the guide.

Reply:
Couldn't find an appropriate translation for munging _anywhere_